avae/train.py

```diff
@@ -339,7 +356,27 @@ def train(
                 + str(pose_dims)
                 + ".pt"
             )
-            torch.save(vae, os.path.join("states", mname))
+
+            print(
+                "\n################################################################",
+                flush=True,
+            )
+            print(
+                "Saving model state for restarting and evaluation ...\n",
+                flush=True,
+            )
+
+            torch.save(
+                {
+                    "epoch": epoch + 1,
+                    "model_state_dict": vae.state_dict(),
+                    "optimizer_state_dict": optimizer.state_dict(),
+                    "t_loss_history": t_history,
+                    "v_loss_history": v_history,
+                },
+                os.path.join("states", mname),
```

Comment:
`mname` currently is giving an output like: `avae_20230602_15/11/41_E19_32_6.pt`, it would be better to replace `/` by `-` because the first is confusing to give as an input.

Reply:
@marjanfamili to fix this

Reply:
Fixed in [d6fb7ee](https://github.com/alan-turing-institute/affinity-vae/pull/127/commits/d6fb7eeb2c8cc671b58fdea266de713d1da3b62d)